core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/AbstractSequencerPartitionContainer.java

```diff
@@ -196,7 +196,6 @@ public void setRuns(Collection<Run> runs) {
   public Run getLastRun() {
     Run lastRun = null;
     for (Run thisRun : getRuns()) {
-      if (thisRun == null) continue;
       if (lastRun == null) {
         lastRun = thisRun;
       } else if (thisRun.getStatus() != null && thisRun.getStatus().getStartDate().after(lastRun.getStatus().getStartDate())) {
```

Comment:
startDate may also be null on `thisRun` and/or `lastRun`, which is actually valid and should not result in NPE's